lib/pinpoint_supported_countries.rb

```diff
@@ -77,7 +77,7 @@ def sms_support
       .map do |sms_config|
         iso_code = sms_config['ISO code']
         supports_sms = case trim_spaces(sms_config['Supports Sender IDs'])
-        when 'Registration required1'
+        when 'Registration required1', 'Registration required9'
```

Comment:
I don't read the footnotes as being the same. Footnote 1 seems to be that a Sender ID is required (and must be registered). I read Footnote 9 as _if_ you want to use a Sender ID, it must be registered.

Reply:
ah, so maybe we cross-reference it with `SENDER_ID_COUNTRIES`?